src/main/java/org/assertj/core/util/Files.java

```diff
@@ -82,7 +82,10 @@
    * 
    * @return the system's temporary directory.
    * @throws RuntimeException if this method cannot find or create the system's temporary directory.
+   *
+   * @deprecated For JUnit4, use TemporaryFolder and @Rule; for JUnit5, use TempDirectory extension and @TempDir instead.
```

Comment:
I would refer to the rule/annotation directly: 

```suggestion
   * @deprecated Use either {@link org.junit.jupiter.api.io.TempDir} or {@link org.junit.rules.TemporaryFolder}
```

Reply:
No problem. I will change that.